source/tests/pt/model/test_dpa1.py

```diff
@@ -92,37 +82,9 @@ def test_consistency(
                 atol=atol,
                 err_msg=err_msg,
             )
-            # dp impl
-            dd2 = DPDescrptDPA1.deserialize(dd0.serialize())
-            rd2, _, _, _, _ = dd2.call(
-                self.coord_ext,
-                self.atype_ext,
-                self.nlist,
-            )
-            np.testing.assert_allclose(
-                rd0.detach().cpu().numpy(),
-                rd2,
-                rtol=rtol,
-                atol=atol,
-                err_msg=err_msg,
-            )
-            # dp impl serialization
-            dd3 = DPDescrptDPA1.deserialize(dd2.serialize())
-            rd3, _, _, _, _ = dd3.call(
-                self.coord_ext,
-                self.atype_ext,
-                self.nlist,
-            )
-            np.testing.assert_allclose(
-                rd0.detach().cpu().numpy(),
-                rd3,
-                rtol=rtol,
-                atol=atol,
-                err_msg=err_msg,
-            )
```

Comment:
this test should be place in the common/dpmodel subfolder of the test, not removed!